pkg/generator/caches.go

```diff
@@ -135,17 +134,23 @@ func (caches *Caches) ToEnvoySnapshot(ctx context.Context) (*cache.Snapshot, err
 	defer caches.mu.Unlock()
 
 	localVHosts := make([]*route.VirtualHost, 0, len(caches.translatedIngresses)+1)
+	localTLSVHosts := make([]*route.VirtualHost, 0, len(caches.translatedIngresses)+1)
```

Comment:
We could leave it as a nil slice here in case there are no local tls ingresses (thus no need to have empty slices in tests) but no strong preference as there is no performance diff anyway.

Reply:
Good point, I just went with the same approach as with the other initialisations.